tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -1335,17 +1334,17 @@ static void VerifyPdbUtil(
   {
     CComBSTR str;
     VERIFY_SUCCEEDED(pPdbUtils->GetTargetProfile(&str));
-    VERIFY_ARE_EQUAL(str, L"ps_6_0");
+    VERIFY_ARE_EQUAL(0, wcscmp(str.m_str, L"ps_6_0"));
```

Comment:
We defined a `VERIFY_ARE_EQUAL_STR` for string comparison case in `HlslTestUtils.h`.  `Ref` should be on left, `Src` on right (reversed from order here). That way, `Src` string will be logged when comparison fails.

Reply:
Updated.